docs/API/v3/search-autocomplete-service-resource.md

```diff
@@ -0,0 +1,154 @@
+```
+
+### Request parameters
+
+Name        | In     | Type    | Required | Notes
+----------- | ------ | ------- | -------- | -----
+q           | URL    | string  | no       | The string to compare against package IDs
+skip        | URL    | integer | no       | The number of results to skip, for pagination
+take        | URL    | integer | no       | The number of results to return, for pagination
+prerelease  | URL    | boolean | no       | `true` or `false` determining whether to include [pre-release packages](../../create-packages/prerelease-packages.md)
+semVerLevel | URL    | string  | no       | A SemVer 2.0.0 version string 
+
+The autocomplete query `q` is parsed in a manner that is defined by the server implementation. nuget.org supports
+querying for the prefix of package ID tokens, which are pieces of the ID produced by spliting the original by camel
+case and symbol characters.
+
+The `skip` parameter defaults to 0.
+
+The `take` parameter should be an integer greater than zero. The server implementation may impose a maximum value.
+
+If `prerelease` is not provided, pre-release packages are excluded.
+
+The `semVerLevel` query parameter is used to opt-in to SemVer 2.0.0 packages. If this query parameter is excluded, only
+package IDs with SemVer 1.0.0 compatible versions will be returned. If `semVerLevel=2.0.0` is provided, both SemVer
+1.0.0 and SemVer 2.0.0 compatible packages will be returned. See the
+[SemVer 2.0.0 protocol](https://github.com/NuGet/Home/wiki/Semver-2.0.0-Protocol) for more information.
+
+### Response
+
+The response is JSON document containing up to `take` autocomplete results.
+
+Autocomplete results returned by nuget.org looks something like this, when using `q=storage&prerelease=true`:
+
+[!code-JSON [autocomplete-id-result.json](./_data/autocomplete-id-result.json)]
+
+The root JSON object has the following properties:
+
+Name      | Type             | Required | Notes
+--------- | ---------------- | -------- | -----
+totalHits | integer          | yes      | The total number of matches, disregarding `skip` and `take`
+data      | array of strings | yes      | The package IDs matched by the request
+
+## Enumerate package versions
+
+Once a package ID is discovered using the previous API, a client can use the autocomplete API to enumerate package
+versions for a provided package ID.
+
+An package version that is unlisted will not appear in the results.
+
+```
+GET https://example/autocomplete?id={ID}&prerelease={PRERELEASE}&semVerLevel={SEMVERLEVEL}
+```
+
+### Request parameters
+
+Name        | In     | Type    | Required | Notes
+----------- | ------ | ------- | -------- | -----
+id          | URL    | string  | no       | The package ID to fetch versions for
+prerelease  | URL    | boolean | no       | `true` or `false` determining whether to include [pre-release packages](../../create-packages/prerelease-packages.md)
+semVerLevel | URL    | string  | no       | A SemVer 2.0.0 version string 
```

Comment:
See previous comment on `semVerLevel` for autocomplete of package IDs.

Reply:
Fixed.